{
    "problem_identifier": "problem_42628647-4b5e-4c70-9ab5-151b5a6bbe30",
    "prompt_identifier": "42628647-4b5e-4c70-9ab5-151b5a6bbe30",
    "model_identifier": "gpt-4",
    "score": 0.6666666666666666,
    "sub_criteria_scores": null,
    "issues": [
        "Test failed:\n\tInput: n = [1, 2, 3, 4, -5, 6, 7, 8, 9, 10]; Expected Output: 2\n\tFunction prototype: solve(n: List[int]) -> int\n\tExpected result: 2 <class 'int'>\n\tActual result: 1 <class 'int'>"
    ]
}